lib/models/movie_category.dart

```diff
@@ -0,0 +1,28 @@
+import 'genre.dart';
+import 'movie.dart';
+
+class MovieCategory {
+  final List<Genre> genres;
+  final List<Movie> movies;
+  static Map<int, List<Movie>> mapMovies = {};
+
+  MovieCategory({
```

Comment:
The idea to this class is to separated every page that you can get from the api by a genre?

Reply:
i removed it